e2e/package-lock.json

```diff
@@ -34,10 +34,10 @@
       "integrity": "sha512-nffhOpkymDECQyR0mnsUtoCE8RlX38G0rYP+wgLWFyZuUyuuojSSvi/+euOiQBIn63whYwYVIIH1TvE3tu4OEg==",
       "dev": true,
       "requires": {
-        "fast-deep-equal": "^2.0.1",
```

Comment:
Is it expected changes?

Reply:
Nevermind to this file. Dependencies declared in the "package.json" file. If it has not been changed - ok.

Reply:
I'm not sure about that. I'm not sure about that. I need to clarify it with @Ohrimenko1988.